src/prefect/flows.py

```diff
@@ -827,6 +828,7 @@ def serve(
             paused: Whether or not to set this deployment as paused.
             schedules: A list of schedule objects defining when to execute runs of this deployment.
                 Used to define multiple schedules or additional scheduling options like `timezone`.
+            global_deployment_limit: The maximum number of runs using the deployment created that can run concurrently.
```

Comment:
I think `global_deployment_limit` could be shorted to `global_limit` for brevity while still being clear. Also: here's a suggestion for the docstring:
```suggestion
            global_limit: global_limit: The maximum number of concurrent runs allowed across all served flow instances associated with the same deployment.
```